File: Npc/11000352.cs
using Maple2.Model.Enum;
using Maple2.Script.Npc;

/// <summary>
/// 11000352: Steve
/// </summary>
public class _11000352 : NpcScript {
    protected override int First() {
        return 40;
    }

    // Select 0:
    // $script:0831180407001463$
    // - Can I help you?
    protected override int Select() => 0;

    protected override int Execute(int selection) {
        switch (Id, Index++) {
            case (40, 0):
                // $script:0831180407001466$
                // - You can't step on flowers like that! Don't lie to me, I saw you!
                return -1;
        }
        
        return default;
    }

    protected override NpcTalkButton Button() {
        return (Id, Index) switch {
            (40, 0) => NpcTalkButton.Close,
            _ => NpcTalkButton.None,
        };
    }
}

File: Npc/11000352.py
""" 11000352: Steve """
from npc_api import Option, Script
import random


class Main(Script):
    def first(self) -> int:
        return 40

    def select(self) -> int:
        return 0

    def __0(self, pick: int) -> int:
        # $script:0831180407001463$
        # - Can I help you?
        return None # TODO

    def __40(self, pick: int) -> int:
        # $script:0831180407001466$
        # - You can't step on flowers like that! Don't lie to me, I saw you!
        return -1

    def button(self) -> Option:
        if (self.state, self.index) == (40, 0):
            return Option.CLOSE
        return Option.NONE
